client/src/index.css

```diff
@@ -44,6 +44,11 @@ form {
 }
 
 .form {
-  padding-left: 620px;
   padding-top: 60px;
+  background: #1e081a;
+  max-width: 700px;
+  align-items: center;
+  padding-right: 100px;
+  margin-top: 140px;
+  margin-left: 240px;
```

Comment:
Are you sure you want every form in your app to have these styles, or are these means for a specific form?

Reply:
moved them to the addForm.js